docs/auth.md

```diff
@@ -258,61 +371,63 @@ credentials are then used to authenticate when retrieving any
      - name: basic-user-pass
    ```
 
-1. Then use that `ServiceAccount` in your `TaskRun` (in `run.yaml`):
+1. In `run.yaml`, associate the `ServiceAccount` with your `Run` by doing one of the following:
 
-   ```yaml
-   apiVersion: tekton.dev/v1beta1
-   kind: TaskRun
-   metadata:
-     name: build-push-task-run-2
-   spec:
-     serviceAccountName: build-bot
-     taskRef:
-       name: build-push
-   ```
+   - Associate the `ServiceAccount` with your `TaskRun`:
 
-1. Or use that `ServiceAccount` in your `PipelineRun` (in `run.yaml`):
+     ```yaml
+     apiVersion: tekton.dev/v1beta1
+     kind: TaskRun
+     metadata:
+       name: build-push-task-run-2
+     spec:
+       serviceAccountName: build-bot
+       taskRef:
+         name: build-push
+     ```
 
-   ```yaml
-   apiVersion: tekton.dev/v1beta1
-   kind: PipelineRun
-   metadata:
-     name: demo-pipeline
-     namespace: default
-   spec:
-     serviceAccountName: build-bot
-     pipelineRef:
+   - Associate the `ServiceAccount` with your `PipelineRun`:
+
+     ```yaml
+     apiVersion: tekton.dev/v1beta1
+     kind: PipelineRun
+     metadata:
        name: demo-pipeline
-   ```
+       namespace: default
+     spec:
+       serviceAccountName: build-bot
+       pipelineRef:
+         name: demo-pipeline
+     ```
 
 1. Execute the `Run`:
 
    ```shell
    kubectl apply --filename secret.yaml serviceaccount.yaml run.yaml
    ```
 
-When the `Run` executes, before steps execute, a `~/.docker/config.json` will be
-generated containing the credentials configured in the `Secret`, and these
-credentials are then used to authenticate when retrieving any
-`PipelineResources`.
+## Configuring `docker*` authentication for Docker
```

Comment:
Why the * in docker*?

Reply:
Because the section covers two Secrets called `dockercfg` and `dockerconfigjson`, hence `docker*`. Writing both names out would make the section title really long.